Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.8
```

Comment:
I think there is 3.9 by now but no big deal.

Reply:
YOLO